src/config.rs

```diff
@@ -400,6 +400,9 @@ pub(crate) struct MajorChangeConfig {
     pub(crate) meeting_label: String,
     /// This label signals there are concern(s) about the proposal.
     pub(crate) concerns_label: Option<String>,
+    /// An optional duration for the waiting period after second for the
```

Comment:
Would be nice to specify the unit here. It is days, right?

Reply:
yes, it's in days